src/rust/ide/lib/analytics/src/remote_log.rs

```diff
@@ -9,27 +9,44 @@ pub use wasm_bindgen::prelude::*;
 
 
 #[wasm_bindgen(inline_js="
-export function _remote_log(msg, data) {
-    if (window !== undefined && window.enso !== undefined && window.enso.remote_log !== undefined) {
-        window.enso.remote_log(msg,{data:data})
+export function _remote_log(msg, value) {
+    if (window !== undefined && window.enso !== undefined && window.enso.remoteLog !== undefined) {
+        try {
+            if (value === undefined && value !== null) {
+                window.enso.remoteLog(msg)
+            } else {
+                window.enso.remoteLog(msg,value)
+            }
+        } catch (error) {
+            console.error(\"Error while logging message. \" + error );
+        }
     } else {
         console.warn(\"Failed to send log message.\")
     }
 }
+
+export function _remote_log_value(msg, field_name, value) {
+    const data = {}
+    data[field_name] = value
+    _remote_log(msg,data)
+}
 ")]
 extern "C" {
-     #[allow(unsafe_code)]
-     fn _remote_log(msg:JsValue,data:JsValue);
+    #[allow(unsafe_code)]
+    fn _remote_log_value(msg:JsValue,field_name:JsValue,value:JsValue);
+    #[allow(unsafe_code)]
+    fn _remote_log(msg:JsValue,value:JsValue);
```

Comment:
btw, why do we keep names with `_` on the beginning? We could just keep them in a local `js` module instead.